From 4b4627d86d03300e3a9dc83ec43c485bd100654a Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:45 -0400
Subject: [scsi] qla2xxx: Include <delay.h> file for msleep declartion in qla_nx2.c file

Message-id: <1408391817-6389-58-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87967
O-Subject: [PATCH RHEL 7.1 e-stor 57/69] qla2xxx: Include <delay.h> file for msleep declartion in qla_nx2.c file.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Atul Deshmukh <atul.deshmukh@qlogic.com>

scsi commit id 50256357aa56a6a8e7f6048d79daa8eeb5d7120b

Signed-off-by: Atul Deshmukh <atul.deshmukh@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_nx2.c b/drivers/scsi/qla2xxx/qla_nx2.c
index a8e2603..26de177 100644
--- a/drivers/scsi/qla2xxx/qla_nx2.c
+++ b/drivers/scsi/qla2xxx/qla_nx2.c
@@ -6,6 +6,7 @@
  */
 
 #include <linux/vmalloc.h>
+#include <linux/delay.h>
 
 #include "qla_def.h"
 #include "qla_gbl.h"
-- 
1.7.1